src/node_crypto.cc

```diff
@@ -813,8 +813,8 @@ void SecureContext::SetDHParam(const FunctionCallbackInfo<Value>& args) {
     return env->ThrowError("DH parameter is less than 1024 bits");
   } else if (size < 2048) {
     args.GetReturnValue().Set(FIXED_ONE_BYTE_STRING(
-        env->isolate(), "WARNING: DH parameter is less than 2048 bits"));
-  }
+         env->isolate(), "WARNING: DH parameter is less than 2048 bits"));
+   }
```

Comment:
Btw, `git reset origin/master src/node_crypto.cc` could help in resetting just that file :)